src/Uno.UWP/ApplicationModel/Calls/PhoneCallHistoryManager.Android.cs

```diff
@@ -0,0 +1,102 @@
+﻿

```

Comment:
```suggestion
#nullable enable
﻿
```

Reply:
It cannot be null, as RequestStoreAsyncTask returns PhoneCallHistoryStore not `PhoneCallHistoryStore?`, and sometimes it returns null.

Reply:
ok